src/test/java/org/junitpioneer/testkit/assertion/PropertiesAssert.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2016-2022 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.testkit.assertion;
+
+import java.lang.reflect.Field;
+import java.util.Properties;
+
+import org.assertj.core.api.AbstractAssert;
+import org.junit.platform.commons.support.HierarchyTraversalMode;
+import org.junit.platform.commons.support.ReflectionSupport;
+
+/**
+ * Allow comparisons of java.util.Properties with optional awareness of their structure,
+ * rather than just treating them as Maps.  Object values, which are marginally supported
+ * by Properties, is supported in assertions.
+ *
```

Comment:
```suggestion
 * Allows comparison of {@link java.util.Properties} with optional awareness of their structure,
 * rather than just treating them as Maps. Object values, which are marginally supported
 * by {@code Properties}, is supported in assertions.
 *
```